ui/desktop/src/components/MoreMenu.tsx

```diff
@@ -264,6 +276,24 @@ export default function MoreMenu() {
             >
               Reset Provider
             </button>
+            <button
+              onClick={() => {
+                handleDeleteKeychainKeys()
+                  .then(() => {
+                    console.log('All keys processed.');
+                    toast.success('Deleted provider keys');
+                    setOpen(false);
+                  })
+                  .catch((error) => {
+                    console.error('An error occurred:', error);
+                    toast.error(`An error occurred deleting provider keys ${error}`);
+                    setOpen(false);
+                  });
+              }}
+              className="w-full text-left p-2 text-sm hover:bg-bgSubtle transition-colors text-red-400"
+            >
+              Delete Provider Keys from Keychain
```

Comment:
From a product perspective this feels like something to me that should be buried deeper in the provider settings screen vs the MoreMenu. MoreMenu should be things applicable to nearly all users and things that need to be done frequently.

What would you think about moving it there?